packages/preview/ucpc-solutions/0.1.0/template/main.typ

```diff
@@ -0,0 +1,94 @@
+#import "/lib/lib.typ" as ucpc
```

Comment:
For this import to work when people will create their project from your template, it should use the package name.

```suggestion
#import "@preview/ucpc-solutions:0.1.0" as ucpc
```